lib/plugin/github/GitHub.js

```diff
@@ -32,6 +32,14 @@ class GitHub extends Release {
 
   async init() {
     await super.init();
+
+    // If we're running on GitHub Actions, we can skip the authentication and
+    // collaborator checks.
+    if (this.isGitHubAction) {
```

Comment:
If we basically put `process.env.GITHUB_ACTION` right here, we don't need to add anything to the `GitBase` class (which is also the base class for the Git and GitLab plugins), while keeping our tests more clean/real. In the test for the Github Action you can actually set this environment variable to verify it works properly based on the actual "trigger" (the env var, not the option), keeping the others tests unaffected.

Reply:
Good point! I will have to manually set that env var to be undefined for all the other tests, though since they will in fact be running on GitHub Actions 😂

Reply:
Ah LOL, indeed

Reply:
Forgot to mention, here is where other env vars are (un)set: https://github.com/release-it/release-it/blob/master/test/util/setup.js